src/react-components/settings-menu.js

```diff
@@ -215,22 +217,33 @@ export default class SettingsMenu extends Component {
                 </a>
               </div>
             </div>
-            {showRoomSettings ? (
+            {this.props.scene.is("entered") && showRoomSettings ? (
```

Comment:
you'll want to check `this.props.hubChannel.can("kick_users")` to soft-prevent non-mods from going into streaming mode